src/app/app.component.ts

```diff
@@ -1,11 +1,101 @@
 import { Component } from '@angular/core';
+import { Observable, timer, Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
+  seconds = 0;
+  minutes = 0;
+  hours = 0;
+  buttonStatus = 'Start';
+  isRunning = false;
+  waitStatus = false;
+  showSeconds = '00';
+  showMinutes = '00';
+  showHours = '00';
```

Comment:
https://angular.io/api/common/DecimalPipe
this pipe can help you to get rid of these variables